docs/make.jl

```diff
@@ -20,7 +20,8 @@ const OUTPUT_DIR   = joinpath(@__DIR__, "src/literated")
 to_be_literated = [
     "single_column_os_papa_simulation.jl",
     "one_degree_simulation.jl",
-    "near_global_ocean_simulation.jl"
+    "near_global_ocean_simulation.jl",
+    "coupled_simulation.jl",
```

Comment:
```suggestion
    "atmosphere_ocean_simulation.jl",
```

"coupled_simulation" isn't descriptive enough, since we have ocean, sea ice, atmosphere, land components (and more).